modules/acs-architecture-overview.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * architecture/index.adoc
+
+:_module-type: REFERENCE
+[id="acs-architecture_{context}"]
+= {product-title} architecture
+
+{product-title} installs as pods in your {ocp} or Kubernetes cluster.
+It includes the following components:
+
+|===
+| Component | Description | Quantity
+
+| _Central_
```

Comment:
It's not critical, but I think normally these wouldn't need to be italicized in a table like this.

Reply:
Thanks @bergerhoffer I'll update these.